### DISCUSSION TITLE
Using a Philips Wiz Connected Bulb

### DISCUSSION THREAD

**Discussion by andyman60** [isaacrlevin/presencelight#845] [Q&A] [answered]
Just got a Wiz Connected Philips bulb, and I'm trying to get my Teams status to match... I've got the app installed from the Microsoft store, but it doesn't look like there's any option under the "Configure Lights" for the Wiz type of Philips Bulbs. Maybe I'm missing something? 
![PL](https://github.com/isaacrlevin/presencelight/assets/13338845/6e34753a-ae34-4daa-a2fb-29c62b0020ae)

**Comment by isaacrlevin** [comment]
There are issues with the package dependency I have that controls Wiz lights. I will give it a try and see if it is still happening

**Comment by isaacrlevin** [comment, marked as answer]
Closing as this should be an issue
